draft-ietf-mls-protocol.md

```diff
@@ -3249,32 +3249,42 @@ struct {
 ~~~
 
 On receiving an MLSMessageContent containing a Proposal, a client MUST verify the
-signature inside MLSMessageAuth.  If the signature verifies
-successfully, then the Proposal should be cached in such a way that it can be
-retrieved by hash (as a ProposalOrRef object) in a later Commit message.
+signature inside MLSMessageAuth and that the `epoch` field of the enclosing
```

Comment:
We should delete these requirements here, and make them generic. It seems like you should always check the signature on MLSMessageContentAuth before using the content, regardless of the content type.

Reply:
Thanks for pointing this out! (We used to have a earlier version with more cases and more steps.)